source/common/config/rds_json.cc

```diff
@@ -224,10 +224,6 @@ void RdsJson::translateRoute(const Json::Object& json_route, envoy::api::v2::rou
 
   JSON_UTIL_SET_BOOL(json_route, *match, case_sensitive);
 
-  if (json_route.hasObject("runtime")) {
```

Comment:
I don't think this is right. I think you need to translate to runtime fraction. Please make sure there is also a test that covers this.